Traits/TestsTraits/PhpUnit/TestsRequestHelperTrait.php

```diff
@@ -74,7 +74,7 @@
      * @param array $data
      * @param array $headers
      *
-     * @return  mixed
+     * @return  mixed|\Illuminate\Foundation\Testing\TestResponse
```

Comment:
the `mixed` should be removed, right?
Furthermore, can you add the `@throws UndefinedMethodException` as well?

Reply:
Yes, you are right, will do tomorrow. :) thanks!

Reply:
no problem - you're welcome..